### DISCUSSION TITLE
subtitles/ui affected by glsl shader

### DISCUSSION THREAD

**Discussion by geextahslex** [mpv-player/mpv#14291] [Q&A]
Hi, I use HDR tonemapping shaders called "hdr-toys". I don't know why they are affecting the subtitles and the ui.
Can something be done to fix that?

Thank you :)

normal sdr
![subtitle](https://github.com/mpv-player/mpv/assets/139760266/cfe1a0c1-096b-4561-9a9e-3dd69e76f738)

hdr tonemapping
![subtitle1](https://github.com/mpv-player/mpv/assets/139760266/5ab9a7c9-fa10-40f6-9635-e1e81527beca)

**Comment by ruihe774** [comment]
Did you [`--blend-subtitles`](https://mpv.io/manual/master/#options-blend-subtitles)?

**Reply by kasper93** [reply]
[`--blend-subtitles`](https://mpv.io/manual/master/#options-blend-subtitles) will not change anything for gpu-next. Subtitles are not blended either way, only target rect changes and when they are drawn.

**Comment by kasper93** [comment]
Subtitles are not passed through shaders and if you disable tonemapping by libplacebo they won't be affected.

**Reply by geextahslex** [reply]
>if you disable tonemapping by libplacebo they won't be affected.

Sorry I don't understand. So what am I supposed to do? ^^'

**Reply by natural-harmonia-gropius** [reply]
> disable tonemapping by libplacebo

How? https://libplacebo.org/options/#color-mapping says: 

> Disabling this option does not disable color mapping, it just means "use the default options for everything".

**Reply by haasn** [reply]
^ we should add a "clip/noop" preset

but basically `gamut_mapping=clip,tone_mapping=clip` should disable it

**Reply by geextahslex** [reply]
I already use `tone-mapping=clip gamut-mapping-mode=clip`
This doesn't change anyhting
